tensorrt_llm/_torch/auto_deploy/transform/library/fuse_swiglu.py

```
@@ -0,0 +1,618 @@
+            delete_all_unused_submodules(gm)
+
+        info = TransformInfo(
+            skipped=False, num_matches=cnt, is_clean=cnt == 0, has_valid_shapes=cnt == 0
+        )
+
+        return gm, info
+
+
+# ── NVFP4 quantized SwiGLU pattern matching and fusion ──────────────────────
+
+from ...custom_ops.linear.swiglu import torch_nvfp4_swiglu_mlp  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)